src/main/java/net/starschema/clouddb/jdbc/BQStatementRoot.java

```diff
@@ -240,6 +241,73 @@ public boolean execute(String arg0, String[] arg1) throws SQLException {
         throw new BQSQLException("Not implemented." + "executeBatch()");
     }
 
+    /**
+     * Execute DML (DELETE, INSERT, UPDATE). If you want to perform a SELECT or other DDL
+     * look at {@link #execute(String)}
+     */
+    private int executeDML(String sql) throws SQLException {
+        if (this.isClosed()) {
+            throw new BQSQLException("This Statement is Closed");
+        }
+        if (!connection.shouldUseQueryApi()) {
```

Comment:
For simplicity i only added this feature for the query API.
The documentation in the driver seems like it's not sensible to use the alternate form anyways any longer.

Reply:
Removed!